### ISSUE TITLE
Error on server for wrong username/password

### ISSUE THREAD

**Issue by michaeljoser** [fnakstad/angular-client-side-auth#2]
Hello,

just noticed that if i enter a wrong username/password i get the following error on server console:

```
Error: failed to serialize user into session
    at pass (/home/sites/angular-client-side-auth/node_modules/passport/lib/passport/index.js:267:19)
    at /home/sites/angular-client-side-auth/node_modules/passport/lib/passport/index.js:271:36
    at serializeUser (/home/sites/angular-client-side-auth/models/User.js:66:5)
    at pass (/home/sites/angular-client-side-auth/node_modules/passport/lib/passport/index.js:271:7)
    at Passport.serializeUser (/home/sites/angular-client-side-auth/node_modules/passport/lib/passport/index.js:275:5)
    at IncomingMessage.req.login.req.logIn (/home/sites/angular-client-side-auth/node_modules/passport/lib/passport/http/request.js:43:29)
    at /home/sites/angular-client-side-auth/routes.js:21:21
    at allFailed (/home/sites/angular-client-side-auth/node_modules/passport/lib/passport/middleware/authenticate.js:87:18)
    at attempt (/home/sites/angular-client-side-auth/node_modules/passport/lib/passport/middleware/authenticate.js:231:28)
    at Context.delegate.fail (/home/sites/angular-client-side-auth/node_modules/passport/lib/passport/middleware/authenticate.js:226:9)
```

And if i press the LOGIN multiple times forever seems to stop the script.

Any idea please?

**Comment by Kahramon** [comment]
Hello, 

just a note to the topic: I have the same output, but I thought this is ok as passportjs gives this error message when a user id is undefined and could not serialize ... not sure :-)

**Comment by fnakstad** [comment]
Thanks for catching this! Turns out the /login route didn't return if no user match was found. Fix in commit  e2df3c74c57d61ff12471b762b7b8cd78d7c760a
